crates/typst/src/model/label.rs

```diff
@@ -40,13 +43,13 @@ impl Label {
         /// The name of the label.
         name: EcoString,
```

Comment:
Could use `PicoStr` directly here (and add the necessary cast impls in str.rs)

Reply:
Nice catch!